pkg/apis/eventing/v1alpha1/cluster_provisioner_types.go

```diff
@@ -92,6 +101,16 @@ func (ps *ClusterProvisionerStatus) IsReady() bool {
 	return cProvCondSet.Manage(ps).IsHappy()
 }
 
+// MarkProvisionerReady sets the condition that the provisioner is ready to provision backing resource.
+func (ps *ClusterProvisionerStatus) MarkProvisionerReady() {
+	cProvCondSet.Manage(ps).MarkTrue(ClusterProvisionerConditionProvisionerReady)
```

Comment:
I see, you are just trying to use the mark methods, it is fine I think to just send in the `ClusterProvisionerConditionReady`. As written they will always be the same value.